install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
```

Comment:
Auto-cleanup is why I like /tmp. I've never seen a Linux system with a
significantly limited amount of space in /tmp. Drew, have you? I can live
with /var/tmp though if people prefer it.

Also, Kenton, don't you still have symlink attack type problems, where one
user creates a symlink in /var/tmp waiting for another user to set up
Sandstorm, resulting in an arbitrary system file being overwritten with the
Sandstorm bundle and/or keyring? I could try to whip up a proof-of-concept
system DoS exploit with the current install.sh if you need a demo of this;
presumably I would make a symlink to /etc/shadow and have the Sandstorm
tarball overwrite that, and now no one can log in after someone runs the
Sandstorm installer. To make a useful exploit that escalates privileges
would be more challenging, but possibly possible.